Automattic/newspack-plugin
# feat: command to initialize cron job to slowly backfill CAP term data

### All Submissions:


### Changes proposed in this Pull Request:

This cron (and by extension, the backfill script it executes) is crucial for our efforts to improve performance across our network of sites. This cron will slowly backfill CAP author term data on an hourly basis. This PR relies on [this separate PR in Co-Authors Plus](https://github.com/Automattic/Co-Authors-Plus/pull/1060) which creates the new backfill command. Once this data backfill is complete, we can then implement the [CAP Author Archive filter](https://lobby.vip.wordpress.com/2017/11/07/co-authors-plus-global-filter/) to greatly improve author archive page performance.

Closes # .

### How to test the changes in this Pull Request:

1. Run this query and note the number returned: `SELECT COUNT(*) FROM wp_posts WHERE post_type IN ( 'post' ) AND post_status IN ( 'publish' ) AND post_author <> 0 AND ID NOT IN ( SELECT tr.object_id FROM wp_term_relationships tr LEFT JOIN wp_term_taxonomy tt ON tr.term_taxonomy_id = tt.term_taxonomy_id WHERE tt.taxonomy = 'author' GROUP BY tr.object_id ) ORDER BY ID`
2. Run `wp newspack schedule-co-authors-author-term-backfill`
3. Check that the cron job has been adding by running `wp cron event list` and searching for `newspack_cap_author_term_backfill`
4. Force the cron job to run immediately with: `wp cron event run newspack_cap_author_term_backfill`
5. Re-run the query from Step 1, and confirm that the number has gone down.

### Other information:
